### DISCUSSION TITLE
Projects (beta) – Possibility to delete a project

### DISCUSSION THREAD

**Discussion by pattacini** [community/community#5436] [Projects and Issues] [answered]
As of now a beta project can be closed but not deleted as it can be done with standard projects.

Deleting a beta project should be possible, especially now that we're all doing many tests.

**Comment by CodeCox** [comment]
ditto.

**Comment by mestir** [comment]
Un

**Comment by rampd2** [comment]
cross reference to duplicates created after this was created:
- #6813 (has [answer](https://github.com/github/feedback/discussions/6813#discussioncomment-1545558) from a collaborator)
- #7549
- #7099
- #7402

**Comment by sschuberth** [comment]
> as it can be done with standard projects.

Sorry for being off-topic, but: Really? How? AFAIK standard projects can also only be closed.

*Edit:* Nevermind, I found the "delete" option hidden behind the "edit" option of a closed project... doh!

**Reply by esratraining** [reply]
Unfortunately, there appears to be no "edit" option for projects (beta). I can see it in the non-beta version of projects...

**Reply by thiagola92** [reply]
I thought that would be in settings the "delete" option

**Comment by s805310630** [comment]
My god! This is too bad, there should be a delete function. I have OCD!

**Comment by PythonCoderAS** [comment]
Seriously, we should be able to delete. I made a test project that I do not need and don't want it clogging up my UI.

**Reply by umniways** [reply]
+1
I'm in the same situation

**Reply by Hassoud** [reply]
me too -_-

**Comment by alejandrozepeda** [comment]
Yes! We need the ability to delete projects (beta).

**Comment by vkuzin** [comment]
+1

**Comment by eikob** [comment]
+1 
Absolutely essential to be able to delete old stuff permanently.

**Comment by mbadzhev** [comment]
Testing the beta is good, but the option to delete a project has to be available.

**Comment by ot-net** [comment]
This is very annoying, and should be priority. I would never use beta test for projects again until it's fixed. In fact, I ditched the idea of working on my project via Github just because of this, I had concerns for the future of it when it gets great amount of process and not being able to control my data... Plus, you can't even switch your project between beta to stable, this could be another solution... 
You cannot do anything once you submit, the only option you have is making it private but seeing it there even if it's private is annoying anyways.

**Comment by crashvb** [comment]
Is this not implemented on the backend, or just not exposed to users in beta?

**Reply by ot-net** [reply]
just in beta.

**Comment by ExampleWasTaken** [comment]
I'm in the same situation as pretty much everybody else here. Created a test project and now I don't need it anymore and it's clogging up the interface.

**Comment by tinkermakar** [comment]
+1

**Comment by DamianUS** [comment]
+1

**Comment by adhxtry** [comment]
+1
I don't want such feature which comes with **curse of binding**.

**Comment by rachelannelise** [comment]
+100000 !!!

**Comment by Katsute** [comment]
This feature is on the roadmap https://github.com/github/roadmap/issues/483 for Q2 (Apr-Jun)

> ### Summary
> In order to help teams migrate over from projects classic, we have a number of areas where we are working towards feature parity. Following on from our work https://github.com/github/roadmap/issues/287, in progress for this quarter are:
>
> - [ ] Project events are included in issue / pull request timelines
> - [ ] Ability to delete a project

**Comment by niklas-ekholm** [comment]
+1

**Comment by jarvisryan** [comment]
@github-staff is this something currently being looked into? Even having the ability to turn off the BETA Projects altogether would be great, as some people within our team get confused and think project boards have been deleted as they are required to click a drop down in order to see the old projects menu.

**Comment by axvr** [comment, marked as answer]
As of today, it is now possible to delete projects.

https://github.blog/changelog/2022-05-19-the-new-github-issues-may-19th-update/#%E2%9C%82%EF%B8%8F-delete-projects-you-no-longer-need

**Reply by pattacini** [reply]
Superb!
Thank you for hearing our voices!
